lib/jxl/enc_ans.cc

```diff
@@ -531,6 +531,9 @@ StatusOr<size_t> BuildAndStoreANSEncodingData(
 StatusOr<float> ANSPopulationCost(const ANSHistBin* data,
                                   size_t alphabet_size) {
   float cost = 0.0f;
+  if (ANS_MAX_ALPHABET_SIZE <= alphabet_size) {
```

Comment:
alphabet_size > ANS_MAX_ALPHABET_SIZE ? (in particular, equality is fine)

Reply:
ANS_MAX_ALPHABET_SIZE is 256, so for alphabet size 256, we are still good ok.
Changed it.